kafka/README.md

```diff
@@ -40,4 +40,11 @@ $ HOST_IP=$(ifconfig en0 | grep "inet " | cut -d " " -f2)
 $ docker run -d --name zookeeper --network kafka-net zookeeper:3.4
 $ docker run -d --name kafka --network kafka-net -p 9092:9092 --env ZOOKEEPER_IP=$HOST_IP --env KAFKA_ADVERTISED_HOST_NAME=$HOST_IP ches/kafka
 ```
-Connect to the Kafka broker using your host IP by updating the `bootstrap.servers` accordingly.
+Connect to the Kafka broker using your host IP.
+
+## Connecting to the server
+
+Once your containers are up and running, connect to Kafka by passing in your host and port to the Wildfly server with the environment variables `KAFFKA_HOST` and `KAFKA_PORT` respectively:
```

Comment:
This part is confusing to me "connect to Kafka by passing in your host and port". It is the host and the port on which Kafka is running. We can also add that these are the configurations, u set on the previous step "Getting start with Docker" if a docker image is used.